knowlg-automation/terraform/aws/variables.tf

```diff
@@ -0,0 +1,204 @@
+}
+
+#REDIS
+variable "REDIS_CHART" {
+  description = "Redis Instance Running Namespace"
+  default     = "../../helm_charts/redis"
+}
+
+variable "REDIS_NAMESPACE" {
+  description = "Redis Instance Running Namespace"
+  default     = "knowlg-db"
+}
+#FLINK
+variable "flink_release_name" {
+  type        = list(string)
+  description = "Flink helm release name."
+  default     = ["search-indexer", "audit-event-generator", "asset-enrichment", "post-publish-processor", "dialcode-context-updater", "qrcode-image-generator", "video-stream-generator", "audit-history-indexer"]
+  # default     = ["merged-pipeline"]
+}
+
+variable "flink_namespace" {
+  type        = string
+  description = "Flink namespace."
+  default     = "knowlg-job"
+}
+
+variable "flink_chart_path" {
+  type        = string
+  description = "Flink chart path."
+  default     = "../../helm_charts/flink"
+}
+
+variable "flink_chart_install_timeout" {
+  type        = number
+  description = "Flink chart install timeout."
+  default     = 900
+}
+
+variable "flink_create_namespace" {
+  type        = bool
+  description = "Create flink namespace."
+  default     = true
+}
+
+variable "flink_wait_for_jobs" {
+  type        = bool
+  description = "Flink wait for jobs paramater."
+  default     = false
+}
+
+variable "flink_chart_template" {
+  type        = string
+  description = "Flink chart values.yaml path."
+  default     = "../terraform_helm_templates/flink.yaml.tfpl"
+}
+
+variable "flink_kubernetes_service_name" {
+  type        = string
+  description = "Flink kubernetes service name."
+  default     = "asset-enrichment-jobmanager"
+}
+#CONTENT
+variable "CONTENT_CHART" {
+  description = "Content Instance Running Namespace"
+  default     = "../../helm_charts/content"
+}
+
+variable "CONTENT_NAMESPACE" {
+  description = "Content Instance Running Namespace"
+  default     = "knowlg-api"
+}
+
+#SEARCH
+variable "SEARCH_CHART" {
+  description = "Search Instance Running Namespace"
+  default     = "../../helm_charts/search"
+}
+
+variable "SEARCH_NAMESPACE" {
+  description = "Search Instance Running Namespace"
+  default     = "knowlg-api"
+}
+
+#TAXONOMY
+variable "TAXONOMY_CHART" {
+  description = "Taxonomy Instance Running Namespace"
+  default     = "../../helm_charts/taxonomy"
+}
+
+variable "TAXONOMY_NAMESPACE" {
+  description = "Taxonomy Instance Running Namespace"
+  default     = "knowlg-api"
+}
+
+#LEARNING
+variable "LEARNING_CHART" {
+  description = "Learning Instance Running Namespace"
+  default     = "../../helm_charts/learning"
+}
+
+variable "LEARNING_NAMESPACE" {
+  description = "Learning Instance Running Namespace"
+  default     = "knowlg-api"
+}
+#DIAL
+variable "DIAL_CHART" {
+  description = "DIAL Instance Running Namespace"
+  default     = "../../helm_charts/dial"
+}
+
+variable "DIAL_NAMESPACE" {
+  description = "DIAL Instance Running Namespace"
+  default     = "knowlg-api"
+}
+variable "vpc_cidr" {
+    type        = string
```

Comment:
VPC and Subnet CIDR can't be same. Can you check it once?

Reply:
I took the reference from [this]( https://docs.aws.amazon.com/vpc/latest/userguide/subnet-sizing.html )and as there is a single subnet we defined we can have the block range same, I think?

Reply:
Fine 👍  for one subnet, but in future if we want multiple subnets it could cause an issue. Please check this azure script for ref: https://github.com/Sunbird-Knowlg/knowledge-platform/blob/release-5.7.0/knowlg-automation/terraform/azure/variables.tf#L25

Reply:
so maybe I should put the cidr which is the same as in Azure, it would be good right?

Reply:
Yeah you can.